File: primerp^p.py
import sympy 

def checkP():
    number = 4
    while (True):
        if (sympy.isprime(number)):
            print(number)
            y = number ** number
            x = y + 2
            if (sympy.isprime(x)):
                print("this is it! x: ",x," number: ",number)
                break
            else:
                number = number + 1
        else:
            number = number + 1

checkP()

File: primerp^p.js
var bigInt = require("big-integer");


function checkP(){
    let number = 4;
    while (true) {
        if (bigInt(number).isPrime()){
            console.log(number);
            var y = bigInt(number).pow(number);
            var x = bigInt(y).plus(2);
            if (bigInt(x).isPrime()){
                console.log("this is it; x: " + x + " number: " + number);
                break;
            }else{
                number = number + 1;
            }
        }else{
            number = number + 1;
        }
      }
}



checkP();